x-pack/plugins/security_solution/public/flyout/left/components/prevalence_details_prevalence_cell.tsx

```
@@ -0,0 +1,84 @@
+ * 2.0.
+ */
+
+import type { VFC } from 'react';
+import React from 'react';
+import { EuiLoadingSpinner } from '@elastic/eui';
+import {
+  PREVALENCE_DETAILS_PREVALENCE_CELL_LOADING_TEST_ID,
+  PREVALENCE_DETAILS_PREVALENCE_CELL_VALUE_TEST_ID,
+} from './test_ids';
+import { useFetchFieldValuePairWithAggregation } from '../../shared/hooks/use_fetch_field_value_pair_with_aggregation';
+import { TimelineId } from '../../../../common/types';
+import { useFetchUniqueByField } from '../../shared/hooks/use_fetch_unique_by_field';
+
+export interface PrevalenceDetailsPrevalenceCellProps {
+  /**
+   * The field name
+   * */
+  field: string;
+  /**
+   * The field values
+   */
+  values: string[];
+  /**
+   * The scope id
+   */
+  scopeId: string;
+  /**
+   * The aggregation field
+   */
+  aggregationField: string;
+}
+
+/**
+ * Component displaying a value in many PrevalenceDetails table cells. It is used for the fifth and sixth columns,
+ * which displays the prevalence percentage for host.name and user.name fields.
+ */
+export const PrevalenceDetailsPrevalenceCell: VFC<PrevalenceDetailsPrevalenceCellProps> = ({
+  field,
+  values,
+  scopeId,
+  aggregationField,
+}) => {
+  const {
+    loading: aggregationLoading,
+    error: aggregationError,
+    count: aggregationCount,
+  } = useFetchFieldValuePairWithAggregation({
+    field,
+    values,
+    isActiveTimelines: scopeId === TimelineId.active,
+    aggregationField,
+  });
+
+  const {
+    loading: uniqueLoading,
+    error: uniqueError,
+    count: uniqueCount,
+  } = useFetchUniqueByField({ field: aggregationField });
+
+  if (aggregationLoading || uniqueLoading) {
+    return (
+      <EuiLoadingSpinner data-test-subj={PREVALENCE_DETAILS_PREVALENCE_CELL_LOADING_TEST_ID} />
+    );
+  }
+
+  const prevalence = aggregationCount / uniqueCount;
+  if (aggregationError || uniqueError || !isFinite(prevalence)) {
+    return null;
```

again, not sure if hiding the error is the way to go